playbooks/gce/openshift-cluster/launch_instances.yml

```diff
@@ -0,0 +1,39 @@
+
+- set_fact:
+    machine_type: "{{ lookup('env', 'gce_machine_type') |default('n1-standard-1', true) }}"
+    machine_image: "{{ lookup('env', 'gce_machine_image') |default('libra-rhel7', true) }}"
+
+- name: Launch instance(s)
+  gce:
+    instance_names: "{{ instances }}"
+    machine_type: "{{ machine_type }}"
+    image: "{{ machine_image }}"
+    service_account_email: "{{ lookup('env', 'gce_service_account_email_address') }}"
+    pem_file: "{{ lookup('env', 'gce_service_account_pem_file_path') }}"
+    project_id: "{{ lookup('env', 'gce_project_id') }}"
+    tags:
+      - "created-by-{{ cluster }}"
```

Comment:
hmm, created by should always be the username of the person who created it. Otherwise we'll have tags like "created-by-int" and "created-by-prod".

The fact that we like to have each user use their own username for their environment name is just a coincidence.

I'd change this to (untested):
```
- "created-by-{{ lookup('env', 'USER') }}"
```